source/playbooks/interact-with-playbooks.rst

```diff
@@ -0,0 +1,26 @@
+Interact with playbooks
```

Comment:
Can you add the appropriate subscription and deploy badges to this page, please?